.github/renovate.json

```diff
@@ -79,6 +79,7 @@
       }
     }
   ],
+  "postUpdateOptions": ["yarnDedupeHighest"],
```

Comment:
default strategy is highest which is what `yarn deduplicate` is doing right now.